docs/pages/database-access/guides/clickhouse-self-hosted.mdx

```diff
@@ -0,0 +1,250 @@
+  and Teleport Database Access with ClickHouse Native protocol does not include audit logs of database query activity.
+</Admonition>
+
+This guide will help you to:
+
+- Install and configure Teleport.
+- Set up Teleport to access your self-hosted ClickHouse Database.
+- Connect to your database through Teleport.
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+![Teleport Database Access Self-hosted ClickHouse](../../../img/database-access/guides/clickhouse_selfhosted_selfhosted.png)
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+![Teleport Database Access ClickHouse Cloud](../../../img/database-access/guides/clickhouse_selfhosted_cloud.png)
+</ScopedBlock>
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+- For ClickHouse HTTP Protocol integration self-hosted ClickHouse server v22.3 or later.
+- For ClickHouse Native Protocol integration self-hosted ClickHouse Server v23.3 or later.
+- The [clickhouse-client](https://clickhouse.com/docs/en/interfaces/cli) installed and added to your system's `PATH` environment variable
+
+## Step 1/5. Create a Teleport token and user
+
+(!docs/pages/includes/database-access/token.mdx!)
+
+(!docs/pages/includes/database-access/create-user.mdx!)
+
+## Step 2/5. Create a certificate/key pair
+
+(!docs/pages/includes/database-access/tctl-auth-sign.mdx!)
+
+Follow the instructions below to generate TLS credentials for your database.
+
+```code
+# Export Teleport's certificate authority and a generated certificate/key pair
+# for host db.example.com with a 1-year validity period.
+$ tctl auth sign --format=db --host=db.example.com --out=server --ttl=8766h
+```
+
+(!docs/pages/includes/database-access/ttl-note.mdx!)
+
+The command will create three files:
+
+- `server.cas` with Teleport's certificate authority
+- `server.key` with a generated private key
+- `server.crt` with a generated host certificate
+
+## Step 3/5. Configure ClickHouse Server
+
+Use the generated secrets to enable mutual TLS in your `clickhouse-server/config.xml` configuration file:
+
+```xml
+<openSSL>
+    <server>
+       <privateKeyFile>/path/to/server.key</privateKeyFile>
+       <caConfig>/path/to/server.cas</caConfig>
+       <certificateFile>/path/to/server.crt</certificateFile>
+       <verificationMode>strict</verificationMode>
+    </server>
+</openSSL>
+```
+
+See [SSL-TLS](https://clickhouse.com/docs/en/guides/sre/configuring-ssl) ClickHouse documentation for more details.
+
+Additionally, your ClickHouse database user accounts must be configured to require a valid client certificate.
+
+```sql
+CREATE USER alice IDENTIFIED WITH ssl_certificate CN 'alice';
+```
+
+By default, the created user may not have access to anything and won't be able to connect, so let's grant it some permissions:
+
+```sql
+GRANT ALL ON *.* TO alice;
+```
+
+## Step 4/5. Configure and Start the Database Service
+Install and configure Teleport where you will run the Teleport Database Service:
+
+<Tabs>
```

Comment:
I think this `Tabs` component is too complicated. It's difficult for me to follow the layers of nested tab items, dropdown menus, etc. 

To start, I would suggest:
- In the "Linux Server" tab, removing the option to start the Database Service with CLI flags and use only the configuration file tab
- The only difference between the "ClickHouse HTTP protocol" and "ClickHouse native protocol" tab items is the value of one flag/config field. I think we can avoid using tabs for the `teleport db configure create` instructions and tell the reader which field to use.

Reply:
Yea, Agree many nested tabs are quite unreadable. I have reworked it according to your suggestion.